docs/index.asciidoc

```diff
@@ -1,16 +1,28 @@
 :branch: current
+include::../_beats/libbeat/docs/version.asciidoc[]
 include::{asciidoc-dir}/../../shared/attributes.asciidoc[]
 
+:version: {stack-version}
+:beatname_lc: apm-server
+:beatname_uc: APM Server
+:beatname_pkg: {beatname_lc}
+:dockerimage: docker.elastic.co/apm/{beatname_lc}:{version}
+:dockergithub: https://github.com/elastic/apm-server-docker/tree/{doc-branch}
```

Comment:
Did you forget to define `doc-branch` and `stack-version`?

Reply:
They are defined and included from here: https://github.com/elastic/apm-server/blob/master/_beats/libbeat/docs/version.asciidoc

Before we backport this we'll need to override those to match our 6.0

Reply:
ah overlooked that